web-services/handler.js

```diff
@@ -102,7 +102,7 @@ module.exports.getAction = async (event, context) => {
   } 
   //process and return data for chartdb endpoint 
   return {
-    statusCode: 200,
+    statusCode: 500,
```

Comment:
Looks like the intention was missed here. The response status code should be 200 if the operation above was successful, but if there was a failure (ie. it got into the "catch" block), it should return a 500 status code.